ci/cron/daily-compat.yml

```diff
@@ -67,6 +67,26 @@ jobs:
           pathtoPublish: '$(Build.StagingDirectory)'
           artifactName: 'Bazel Compatibility Logs'
 
+  - job: platform_independence_windows
+    timeoutInMinutes: 60
+    pool:
+      name: windows-pool
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - template: ../platform-independence-windows.yml
+
+  - job: platform_independence
```

Comment:
probably a good idea to include macos as well. Not that I see a reason why that should be different but easy enough to add and better to check it than to be surprised later.

Reply:
```suggestion
  - job: platform_independence
    strategy:
      matrix:
        linux:
          pool: ubuntu_20_04
        macos:
          pool: macOS-pool
```